test/Data/Generic/Rep.purs

```diff
@@ -123,6 +123,9 @@ testGenericRep = do
   assert "Checking show" $
     show (cons 1 (cons 2 Nil)) == "(Cons { head: 1, tail: (Cons { head: 2, tail: Nil }) })"
 
+  assert "Checking show for generic types" $
+    show (G.from $ cons 1 Nil) == "(Inr (Constructor \"Cons\" (Argument { head: 1, tail: Nil })))"
+
```

Comment:
Could we also test the show output for `G.from Nil` to cover `NoArguments` and the `Inl` constructor, as well as a constructor with multiple arguments to cover `Product`?

Reply:
Yup. Fixed.